tests/src/test/scala/system/basic/WskRuleTests.scala

```diff
@@ -83,43 +81,6 @@ abstract class WskRuleTests extends TestHelpers with WskTestHelpers {
 
   behavior of "Whisk rules"
 
-  it should "preserve rule status when a rule is updated" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
```

Comment:
Why does this test need to be moved?

Reply:
@dubee I moved this test due to the way the response is evaluated in REST vs the CLI. This was an update that was inconsistent across the "synced" `WskRuleTests` files. When I left this here, in it's current state, I was met with this exception from the CLI repo tests:
```
system.basic.WskCliRuleTests STANDARD_OUT
    Starting test Whisk rules should preserve rule status when a rule is updated at 2018-03-29 17:25:27.372
system.basic.WskCliRuleTests > Whisk rules should preserve rule status when a rule is updated STANDARD_OUT
    Exception occurred during test execution: spray.json.JsonParser$ParsingException: Unexpected character 'o' at input index 0 (line 1, position 1), expected JSON Value:
    ok: updated rule r1to1-1522344327373
    ^
system.basic.WskCliRuleTests > Whisk rules should preserve rule status when a rule is updated FAILED
    spray.json.JsonParser$ParsingException: Unexpected character 'o' at input index 0 (line 1, position 1), expected JSON Value:
    ok: updated rule r1to1-1522344327373
    ^
```
It expects a JSON response, while the CLI returns that string.

Do you think this would be better off merged with https://github.com/apache/incubator-openwhisk-cli/pull/262/files#r178365760 ?

I can try to mash them together and work some evaluation logic to handle the 2 cases if we think that is appropriate.

Reply:
I have merged the tests to use the `cli` boolean flag now that I can override in the CLI extentions